### PR TITLE
Add examples for empty list response and get response

### PR DESCRIPTION
**Pull request by jimmybyrum**

### CODE REVIEW

File: spec/v2/index.bs [hunk 1]
```diff
@@ -2284,6 +2291,34 @@ highlight: json
...
 </div>
 
+### Example Action GetFootprint request and response ### {#api-action-get-example}
+
+<div class="example">
+
+Example request:
+
+<pre highlight=http>
+GET /2/footprint/<l>[=GetPfId=]</l> HTTP/2
```
Comment (zeitgeist) [inline spec/v2/index.bs:2301]: A minor suggestion to make this is a more self-contained example:

```suggestion
GET /2/footprint/d9be4477-e351-45b3-acd9-e1da05e6f633 HTTP/2
```

**Review by zeitgeist — COMMENTED** [review]
@jimmybyrum thank you for your kind contribution. This looks good to me modulo a minor suggestion. 

Leaving this to @bhadley for a final acceptance decision.

**Review by zeitgeist — APPROVED** [review]
Thank you, @jimmybyrum . Looks good to me.

**Review by bhadley — APPROVED** [review]
Great idea to add this example! I don't consider myself qualified to review the technical details here, but want to thank you @jimmybyrum for this contribution!

**Pull request merged**